monai/transforms/croppad/dictionary.py

```diff
@@ -315,7 +316,10 @@ def inverse(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, Nd
         for key in self.key_iterator(d):
             transform = self.get_most_recent_transform(d, key)
             # Create inverse transform
-            orig_size = np.array(transform[TraceKeys.ORIG_SIZE])
+            orig_size = transform[TraceKeys.ORIG_SIZE]
+            if isinstance(orig_size[0], torch.Tensor):
+                orig_size = torch.as_tensor(orig_size)
+            orig_size = np.asarray(orig_size)
```

Comment:
Hi @rijobro ,

I think maybe you can use the `convert_data_type` utility here to convert to numpy array with `wrap_sequence` arg?

Thanks.